src/domain/targeted-messaging/targeted-messaging.repository.ts

```diff
@@ -45,4 +53,8 @@ export class TargetedMessagingRepository
   }): Promise<Submission> {
     return this.datasource.createSubmission(args);
   }
+
+  async getOutreach(args: { outreachId: number }): Promise<Outreach> {
```

Comment:
Nit: If we only have one property inside the object we can define it directly:
`async getOutreach(outreachId: number): Promise<Outreach> {`

Reply:
https://github.com/safe-global/safe-client-gateway/commit/1b72d7acb028af5275397cac1a6b18e63e85e435#diff-14eea5767397a0c69489890e40793e5400cffb4237b1a6cb426652651659ad9bL57